src/main/java/frc/robot/Constants.java

```diff
@@ -15,9 +17,36 @@
 public final class Constants {
   public static final int PLACEHOLDER_INT = 0;
   public static final double PLACEHOLDER_DOUBLE = 0.0;
+  public static final float PLACEHOLDER_FLOAT = 0;
+
 
   public static final int DRIVER_CONTROLLER_PORT = 0;
   public static final int OPERATOR_CONTROLLER_PORT = 1;
 
   public static final int NUM_CANDLE_LEDS = PLACEHOLDER_INT;
+
+  public static final class SwerveModule {
+    public static final boolean TURNING_ENCODER_INVERTED = true;
+
+    /* Meters */
+    public static final double DRIVING_ENCODER_POSITION_FACTOR_METERS = PLACEHOLDER_DOUBLE;
+    /* Meters per second */
+    public static final double DRIVING_ENCODER_VELOCITY_FACTOR_METERS_PER_SECOND = PLACEHOLDER_DOUBLE;
```

Comment:
The logic for these is the same as in the 2023 bot (it's just unit conversions, basically) AND we know the specs of this robot (see pinned doc in software)